app/templates/index.html

```diff
@@ -12,11 +12,11 @@ <h1>
 
 {% block phase_banner %}{% endblock %}
 
- {% block body_classes %}index-page{% endblock %}
+ {% block body_classes %}index-page-body{% endblock %}
```

Comment:
what does this block do? I can't find it anywhere else in the buyer frontend.

Reply:
@Wynndow - https://github.com/alphagov/govuk_template/blob/master/source/views/layouts/govuk_template.html.erb#L39